### PR TITLE
MGMT-19410: Export control_plane_count cluster property

### PR DESCRIPTION
**Pull request by danmanor**
Export control_plane_count cluster property so it will be accessible by clients. The UI needs the value tp present to the user his previous selection. 

## List all the issues related to this PR

- [ ] New Feature 
- [x] Enhancement 
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [x] Automation (CI, tools, etc)
- [x] Cloud
- [x] Operator Managed Deployments
- [ ] None

## How was this code tested?

- [x] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [x] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [ ] No tests needed

## Checklist

- [x] Title and description added to both, commit and PR.
- [x] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [x] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- Are the title and description (in both PR and commit) meaningful and clear?
- Is there a bug required (and linked) for this change?
- Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -5475,7 +5475,9 @@ definitions:
       org_soft_timeouts_enabled:
         type: boolean
         description: Indication if organization soft timeouts is enabled for the cluster.
-
+      control_plane_count:
+        type: integer
+        description: The amount of control planes which should be part of the cluster.
```
Comment (paul-maidment) [inline swagger.yaml:5480]: "The number of control plane nodes required for this cluster"

**Review by paul-maidment — COMMENTED** [review]
Looks generally good, I suggested one small change to one comment.

**Review by paul-maidment — APPROVED** [review]
/lgtm

**Comment by danmanor** [comment]
/override ci/prow/edge-e2e-metal-assisted-osc-4-17 ci/prow/edge-e2e-metal-assisted-osc-sno-4-17

**Comment by danmanor** [comment]
/override ci/prow/edge-e2e-metal-assisted-osc-4-17 ci/prow/edge-e2e-metal-assisted-osc-sno-4-17

**Review by avishayt — APPROVED** [review]

**Pull request merged**